lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/Library.java

```diff
@@ -46,11 +46,13 @@ public void addBook(Book book) {
    * Remove a book from the library.
    *
    * @param book The book to remove.
+   * @throws BookCheckedOutException If the book is checked out.
    */
   public void removeBook(Book book) throws BookCheckedOutException {
```

Comment:
Why keep these methods when the check in/check out functionality can be handled by `checkOutMedia` and `checkInMedia`?

Reply:
I put this comment in the wrong place, I meant to add this to `checkInBook` below.